From f5227c249f03e9aa84f808315be770a16740d1a1 Mon Sep 17 00:00:00 2001
From: Zhou Peng-B04994 <eagle.zhou@nxp.com>
Date: Wed, 25 Oct 2017 09:13:15 +0800
Subject: [PATCH 2679/5242] MLK-16671-5 - [i.MX8QXP/Malone]: Add vpu malone
 decoder driver

commit  239f70d7a2d78b18fa0764da090a13780e1b7023 from
https://source.codeaurora.org/external/imx/linux-imx.git

Change statement from LGPL to GPL for malone header files

Signed-off-by: Zhou Peng-B04994 <eagle.zhou@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/linux/mxc_vpu-malone.h |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/include/linux/mxc_vpu-malone.h b/include/linux/mxc_vpu-malone.h
index 8214157..4541c873 100755
--- a/include/linux/mxc_vpu-malone.h
+++ b/include/linux/mxc_vpu-malone.h
@@ -3,12 +3,12 @@
  */
 
 /*
- * The code contained herein is licensed under the GNU Lesser General
- * Public License.  You may obtain a copy of the GNU Lesser General
- * Public License Version 2.1 or later at the following locations:
+ * The code contained herein is licensed under the GNU General Public
+ * License. You may obtain a copy of the GNU General Public License
+ * Version 2 or later at the following locations:
  *
- * http://www.opensource.org/licenses/lgpl-license.html
- * http://www.gnu.org/copyleft/lgpl.html
+ * http://www.opensource.org/licenses/gpl-license.html
+ * http://www.gnu.org/copyleft/gpl.html
  */
 
 /*!
-- 
1.7.9.5

